Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
>  Either change your commit message to reflect that, or find a right place to make a call to setFormats

Maybe the right place would be the SonataCoreExtension class ? What do you think @Soullivaneuh ?